hw01_hello_otus/go.mod

```
@@ -1,3 +1,5 @@
-module github.com/fixme_my_friend/hw01_hello_otus
+module github.com/VladimirButakov/home-work/tree/master/hw01_hello_otus
```

```suggestion
module github.com/VladimirButakov/hw-test/hw01_hello_otus
```
Рекомендую указать корректный путь до модуля, чтобы он был доступен для скачивания